src/gmres.jl

```diff
@@ -114,7 +114,7 @@ function gmres_method!(log::ConvergenceHistory, x, A, b;
 
         @eval a = $(VERSION < v"0.4-" ? Triangular(H[1:N, 1:N], :U) \ s[1:N] : UpperTriangular(H[1:N, 1:N]) \ s[1:N])
         w = a[1:N] * K
-        @blas! x = solve(Pr,w) #Right preconditioner
+        @blas! x += solve(Pr,w) #Right preconditioner
```

Comment:
Thanks for spotting that! I totally missed it when translating. Could you also add a test case to [this](https://github.com/JuliaMath/IterativeSolvers.jl/blob/master/test/runtests.jl#L59) file? Just checking if it converges correctly.